ci-operator/config/openshift/csi-external-snapshotter/openshift-csi-external-snapshotter-master.yaml

```diff
@@ -44,6 +44,13 @@ images:
       as:
       - registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7
   to: csi-snapshot-controller
+- dockerfile_path: Dockerfile.webhook.openshift.rhel7
```

Comment:
I just realized that we suffix our Dockerfiles with `.rhel7` but we use RHEL8 as base image. Not a big deal, but we might want to fix that in the future...

Reply:
It should be possible, but not so easy - they're recorded somewhere in ART data.